pkg/agent/util/net.go

```diff
@@ -401,13 +401,20 @@ func GenerateUplinkInterfaceName(name string) string {
 	return name + bridgedUplinkSuffix
 }
 
-func GenerateRandomMAC() net.HardwareAddr {
+func GenerateRandomMAC(global bool) net.HardwareAddr {
```

Comment:
All virtual devices should set local assignment bit:
1. All MAC addresses including veth, OVS internal port, OVS datapath ID set it;
2. linux kernel implementation does the same for all virtual devices: https://elixir.bootlin.com/linux/v4.15.18/source/include/linux/etherdevice.h#L227

Reply:
Originally, I hit some failures in the tests when creating network interfaces with a pre-allocated MAC if I only set the local bit, and I thought it was related with the local bit, so I add this global parameter. Since Linux kernel uses this logic, I would update accordingly.